buildpipeline/Core-Setup-Linux-Arm-BT.json

```diff
@@ -242,6 +206,16 @@
     {
       "enabled": false,
```

Comment:
This is just what the VSTS download API returned, it could probably be removed, but it's getting added back whenever the definition is loaded into VSTS.  It would be nice if we could trim these down to the minimum data necessary, but for now we tend to just preserve whatever downloading the VSTS definition gives us.